### DISCUSSION TITLE
Comments are broken

### DISCUSSION THREAD

**Discussion by jcubic** [community/community#11847] [Mobile]
Just noticed this. I've added two comments the 2nd comment had content from the previous comment. So I thought that this was an edit, and what I've ended up with were two comments with the same content and the beginning.

So the bug is that comment input should be cleared after you add a comment.

**Comment by matkoniecz** [comment]
Where you were commenting? Discussion response? Code review? Issue comment?

**Comment by jcubic** [comment]
Sorry, it was an Issue.

On Mon, 21 Feb 2022 02:52:02 -0800
Mateusz Konieczny ***@***.***> wrote:

> Where you were commenting? Discussion response? Code review? Issue comment?
> 

--
Jakub T. Jankiewicz, Web Developer
https://jcubic.pl/me

**Reply by candyho** [reply]
Hey! Sorry for the inconvenience caused. Do you mind telling us which platform are you using (android or iOS)? It'd also be really helpful to let us know the app version and the device model so that we could try to reproduce and debug. Thank you.

**Comment by jcubic** [comment]
Android. I can check again, I'm not sure if I've had updated apps. They don't
update automatically.

On Mon, 21 Feb 2022 10:34:44 -0800
Candy Ho ***@***.***> wrote:

> Hey! Sorry for the inconvenience caused. Do you mind telling us which
> platform are you using (android or iOS)? It'd also be really helpful to let
> us know the app version and the device model so that we could try to
> reproduce and debug. Thank you. 
> 

--
Jakub T. Jankiewicz, Web Developer
https://jcubic.pl/me
